tests/integration/test_render.py

```diff
@@ -8,22 +25,73 @@ class TestDragAndDropRender(BaseIntegrationTest):
     PAGE_TITLE = 'Drag and Drop v2'
     PAGE_ID = 'drag_and_drop_v2'
 
-    def setUp(self):
-        super(TestDragAndDropRender, self).setUp()
-
-        scenario_xml = "<vertical_demo><drag-and-drop-v2/></vertical_demo>"
+    def load_scenario(self, item_background_color="", item_text_color=""):
+        scenario_xml = """
+<vertical_demo>
+    <drag-and-drop-v2 item_background_color='{item_background_color}' item_text_color='{item_text_color}' />
+</vertical_demo>
+        """.format(item_background_color=item_background_color, item_text_color=item_text_color)
         self._add_scenario(self.PAGE_ID, self.PAGE_TITLE, scenario_xml)
 
         self.browser.get(self.live_server_url)
         self._page = self.go_to_page(self.PAGE_TITLE)
 
-    def _test_style(self, item, style_settings):
-        style = item.get_attribute('style')
+    def _test_style(self, element, style_settings, element_type):
+        style = element.get_attribute('style')
         for style_prop, expected_value in style_settings.items():
+            if style_prop == 'color' or style_prop == 'background-color' and expected_value.startswith('#'):
+                expected_value = Colors.rgb(expected_value)
             expected = u"{0}: {1}".format(style_prop, expected_value)
             self.assertIn(expected, style)
+        if element_type == "item":
+            self._test_item_style(element, style_settings, style)
```

Comment:
I think you could do the test for CSS properties a lot more robustly with something like this:

```
bg_color =  self.browser.execute_script('getComputedStyle($("#tester").get(0)).backgroundColor'
```

The advantage of that is you don't have to worry about parsing the `style` string, and it checks the "computed" values, so you can check what color is actually used in the browser regardless of how complex the CSS styles/rules are.